go.mod

```diff
@@ -180,12 +180,12 @@ replace (
 	//
 	// k8s.io/kubernetes depends on these k8s.io packages, but unversioned
 	//
-	k8s.io/api => k8s.io/api v0.26.1
+	k8s.io/api => k8s.io/api v0.26.7
 	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.26.1
-	k8s.io/apimachinery => k8s.io/apimachinery v0.26.1
+	k8s.io/apimachinery => k8s.io/apimachinery v0.26.7
 	k8s.io/apiserver => k8s.io/apiserver v0.26.1
 	k8s.io/cli-runtime => k8s.io/cli-runtime v0.26.1
```

Comment:
Is there a reason to not update all k8s.io packages at once?

Reply:
Yes we can change it as well 👍🏻 as the bug is in the client-go package it's only updated and not sure if updating other dependencies might cause some other regression. @nixpanic whats your suggestion?

Reply:
I guess it is recommended to update to the latest versions when possible. That will include bugfixes and so on. Mixing different versions is mst likely less well tested than using the same version for everything. In case we hit an unidentified regression, we can always downgrade again. I hope that the projects we use have good CI, and regressions are kept to a minimum.

Reply:
Thanks @nixpanic & @Madhu-1, will update all the packages in that case.

Reply:
Done.